lib/tbd/psi.rb

```diff
@@ -1769,6 +1769,50 @@ def process(model = nil, argh = {})
       end
     end
 
+    # Fetch edge multipliers for subsurfaces, if applicable.
+    edges.values.each do |edge|
+      next     if edge.key?(:mult)                    # skip if already assigned
+      next unless edge.key?(:surfaces)
+      next unless edge.key?(:psi)
+      ok = false
+
+      edge[:psi].keys.each do |k|
+        break if ok
+
+        jamb = k.to_s.include?("jamb")
+        sill = k.to_s.include?("sill")
+        head = k.to_s.include?("head")
+        ok   = jamb || sill || head
+      end
+
+      next unless ok     # if OK, edge links subsurface(s) ... yet which one(s)?
+
+      edge[:surfaces].each do |id, surface|
+        next unless tbd[:surfaces].key?(id)    # look up parent (opaque) surface
+
+        [:windows, :doors, :skylights].each do |subtypes|
+          next unless tbd[:surfaces][id].key?(subtypes)
+
+          tbd[:surfaces][id][subtypes].each do |nom, sub|
+            next unless edge[:surfaces].key?(nom)
+            next unless sub[:mult] > 1
+
+            # An edge may be tagged with (potentially conflicting) multipliers.
+            # This is only possible if the edge links 2 subsurfaces, e.g. a
+            # shared jamb between window & door. By default, TBD tags common
+            # subsurface edges as (mild) "transitions" (i.e. PSI 0 W/K.m), so
+            # there would be no point in assigning an edge multiplier. Users
+            # can however reset an edge type via a TBD JSON input file (e.g.
+            # "joint" instead of "transition"). It would be a very odd choice,
+            # but TBD doesn't prohibit it. If linked subsurfaces have different
+            # multipliers (e.g. 2 vs 3), TBD tracks the highest value.
```

Comment:
This will definitely makes it way to a new "subsurface" tab in the online Guide.

Reply:
Do you have a PR for the guide updates?

Reply:
Nope.